### ISSUE TITLE
ARM64 linux do not compile

### ISSUE THREAD

**Issue by neofeo** [royshil/obs-backgroundremoval#334]
Hi, there is something that needs to be patched in order to get it compiled to arm64 linux 
it almost compiled bu..

```
-- Installing: /home/salva/Desktop/obs-backgroundremoval/build/OpenCV_Build-prefix/share/opencv4/lbpcascades/lbpcascade_frontalface.xml
-- Installing: /home/salva/Desktop/obs-backgroundremoval/build/OpenCV_Build-prefix/share/opencv4/lbpcascades/lbpcascade_frontalface_improved.xml
-- Installing: /home/salva/Desktop/obs-backgroundremoval/build/OpenCV_Build-prefix/share/opencv4/lbpcascades/lbpcascade_profileface.xml
-- Installing: /home/salva/Desktop/obs-backgroundremoval/build/OpenCV_Build-prefix/share/opencv4/lbpcascades/lbpcascade_silverware.xml
[ 50%] Completed 'OpenCV_Build'
[ 50%] Built target OpenCV_Build
[ 56%] Building C object CMakeFiles/obs-backgroundremoval.dir/src/plugin-main.c.o
[ 62%] Building CXX object CMakeFiles/obs-backgroundremoval.dir/src/ort-utils/ort-session-utils.cpp.o
[ 68%] Building CXX object CMakeFiles/obs-backgroundremoval.dir/src/obs-utils/obs-utils.cpp.o
[ 75%] Building C object CMakeFiles/obs-backgroundremoval.dir/src/background-filter-info.c.o
[ 81%] Building CXX object CMakeFiles/obs-backgroundremoval.dir/src/background-filter.cpp.o
[ 87%] Building CXX object CMakeFiles/obs-backgroundremoval.dir/src/enhance-filter.cpp.o
[ 93%] Building C object CMakeFiles/obs-backgroundremoval.dir/src/enhance-filter-info.c.o
[100%] Linking CXX shared module obs-backgroundremoval.so
/usr/bin/ld: _deps/onnxruntime-src/lib/libonnxruntime.so.1.15.0: error adding symbols: file in wrong format
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/obs-backgroundremoval.dir/build.make:199: obs-backgroundremoval.so] Error 1
make[1]: *** [CMakeFiles/Makefile2:85: CMakeFiles/obs-backgroundremoval.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
```

**Comment by royshil** [comment]
yeah.. we're not building ONNX runtime from scratch because it takes about 45 minutes.
i'm actually not 100% sure where the onnxruntime .so file is coming from, it's possible your pkg mgr is getting an invalid version... seeing as it doesn't match your computer's architecture.

**Comment by umireon** [comment]
@neofeo Can you try #335 tree?

**Comment by umireon** [comment]
Implemented by #335
